classes/openrc.bbclass

```diff
@@ -97,10 +118,46 @@ python openrc_populate_packages() {
         localdata.prependVar("OVERRIDES", f"{pkg}:")
         localdata.setVar(f"OPENRC_SERVICES:{pkg}", " ".join(services))
 
+        # Prepare per-service runlevel variables and escaped service names for shell scripts
+        script_vars = ""
+        runlevel_vars = ""
+        services_escaped = ""
+
+        for service in services:
+            runlevel = d.getVar(f"OPENRC_RUNLEVEL:{service}")
```

Comment:
Don't we need `service` in `OVERRIDES` for this to work?

Reply:
Not sure it is required as we match exactly the variable OPENRC_RUNLEVEL:service-a which is an existing variable but I will check. From what I understand, the OVERRIDES feature is more a convenient shortcut.

Reply:
Thank you for verifying.